### DISCUSSION TITLE
How to compile an ARM64 version of iPXE?

### DISCUSSION THREAD

**Discussion by GRIFFCOMMca** [ipxe/ipxe#1148] [General]
Hi, we have a UEFI x64 version working, what command can we use to build an ARM64 (UEFI) version of this?  i see an ARCH folder, however cant find any build command for the ARM version in the makefile file.

We found this command "**make bin-arm64-efi/ipxe.efi EMBED=embed.ipxe**", we assumed would compile the same code for ARM, however we are getting errors "**unrecognized command-line option '-mlittle-endian**'

This (virtual) machine compiles the x64 efi version ok (Ubuntu 20 i believe it is)

Many Thanks

**Comment by NiKiZe** [comment]
Searching the repo often gives answers.
#1088 has examples ;)
